plugins/lookup/onepassword.py

```diff
@@ -574,9 +577,14 @@ def __init__(self, subdomain=None, domain="1password.com", username=None, secret
         self._cli = self._get_cli_class()
 
     def _get_cli_class(self):
-        version = OnePassCLIBase.get_current_version()
+        major_version = self.major_version
+
+        if not major_version:
+            version = OnePassCLIBase.get_current_version()
+            major_version = version.split(".")[0]
```

Comment:
While you're at it, this can be optimized a bit:
```suggestion
            major_version = version.split(".", 1)[0]
```